applications/nrf5340_audio/include/zbus_common.h

```diff
@@ -9,6 +9,8 @@
 
 #include <zephyr/bluetooth/audio/audio.h>
 
+#include "le_audio.h"
```

Comment:
New header added. Have you checked the dependencies diagram?

Reply:
I did not, but I did now, and this PR does introduce a dependency that should be updated in the diagram. Is this something that has to be done by tech writers?